lib/expand/join.go

```diff
@@ -390,13 +390,16 @@ func (p *Peer) executePhase(ctx context.Context, opCtx operationContext, phase i
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	progressReporter := dispatcher.NewProgressReporter(ctx, disp, phaseTitle(phase))
```

Comment:
I'm using progress reporter universally to achieve the same progress output behavior as previously with manual phase execution (with periodic updates with `Still executing ...`).